src/imgproc/imgproc.h

```diff
@@ -268,10 +268,10 @@ class Imgproc : public Math {
 
     dst->resize(w2, h2, src->get_channel_m());
 
```

Comment:
It seems strange to me that this can not copy the data, i will fill an issue for this.